news/feature-3753.md

```diff
@@ -0,0 +1,17 @@
+`stats`: create new statistics counter
+
+the new statistics counters:
+- msg_size_max: The current largest message size of the given source/destination.
+- msg_size_avg: The current average message size of the given source/destination.
+- batch_size_max: When batching is enabled, then this shows the current largest batch size of the given source/destination.
+- batch_size_avg: When batching is enabled, then this shows the current average batch size of the given source/destination.
+
+
+- eps_last_1h: The last hour eps value.
+- eps_last_24h: The last day eps value.
+- eps_since_start: Since driver beggining.
```

Comment:
I think "since startup" might be good enough.
(I know, it's currently the first reader/writer's init time, but eventually we can fix that).